### ISSUE TITLE
pip install pyinstaller broken on freebsd

### ISSUE THREAD

**Issue by ThomasWaldmann** [pyinstaller/pyinstaller#1999]
```
(borg-env) [vagrant@freebsd-10 /vagrant/borg/borg]$ pip install pyinstaller
Collecting pyinstaller
  Downloading PyInstaller-3.2.tar.gz (2.8MB)
    100% |████████████████████████████████| 2.8MB 140kB/s 
Requirement already satisfied (use --upgrade to upgrade): setuptools in /vagrant/borg/borg-env/lib/python3.5/site-packages (from pyinstaller)
Installing collected packages: pyinstaller
  Running setup.py install for pyinstaller
    Complete output from command /vagrant/borg/borg-env/bin/python3.5 -c "import setuptools, tokenize;__file__='/tmp/pip-build-yobf0b9p/pyinstaller/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-da_fasgk-record/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/borg/borg-env/include/site/python3.5/pyinstaller:
    running install
    running build
    running build_bootloader
    waf [commands] [options]

    Main commands (example: ./waf build -j4)
      all             :
        Do configure, build, install in one step.
    ...
    waf: error: no such option: --no-lsb
    No precompiled bootloader found. Trying to compile it for you ...
    ERROR: Failed compiling the bootloader. Please compile manually and rerun setup.py

    ----------------------------------------
Command "/vagrant/borg/borg-env/bin/python3.5 -c "import setuptools, tokenize;__file__='/tmp/pip-build-yobf0b9p/pyinstaller/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-da_fasgk-record/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/borg/borg-env/include/site/python3.5/pyinstaller" failed with error code 1 in /tmp/pip-build-yobf0b9p/pyinstaller
```

**Comment by htgoebel** [comment]
Huh? I thought I've fixed this in a6c0de032a4. I moved adding the `--no-lsb`-option into a `if is_linux` branch. Why is `is_linux` set on FreeBSD?

**Comment by ThomasWaldmann** [comment]
That changeset is not in a pypi release yet, so one won't get it via pip install pyinstaller.

**Comment by ThomasWaldmann** [comment]
So, maybe transform this ticket in a "do a release" reminder? :)

**Comment by ericoporto** [comment]
I have the same error using `pip3 install pyinstaller ` in Cygwin under Windows 7. I have gcc installed in Cygwin.

**Comment by ThomasWaldmann** [comment]
@htgoebel BTW, how about a 3.2.1 - I see the milestone is at 100% since quite a while, but no release yet?

At borgbackup, I still use 3.1.1 (somehow I rather skip x.y.0 releases :).

**Comment by retsyo** [comment]
same error in Cygwin 32bits under Win7 64bits. Why I am using Cygwin? Becuase I want to try urwid
$ pip install pyinstaller
    waf: error: no such option: --no-lsb                                        
    ERROR: Failed compiling the bootloader. Please compile manually and rerun setup.py

**Comment by ninapavlich** [comment]
I'm using Cygwin 64 with Windows 10 64, and getting the same error:
waf: error: no such option: --no-lsb
ERROR: Failed compiling the bootloader. Please compile manually and rerun setup.py

**Comment by htgoebel** [comment]
This was solved by the 3.2.1 release and by 3.3 not defaulting to --no-lsb
